src/Button/MeasureButton/MeasureButton.jsx

```diff
@@ -0,0 +1,747 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { isEmpty } from 'lodash';
+
+import OlMap from 'ol/map';
+import OlLayerVector from 'ol/layer/vector';
+import OlSourceVector from 'ol/source/vector';
+import OlCollection from 'ol/collection';
+import OlStyleStyle from 'ol/style/style';
+import OlStyleStroke from 'ol/style/stroke';
+import OlStyleFill from 'ol/style/fill';
+import OlStyleCircle from 'ol/style/circle';
+import OlInteractionDraw from 'ol/interaction/draw';
+import OlObservable from 'ol/observable';
+import OlOverlay from 'ol/overlay';
+
+import ToggleButton from '../ToggleButton/ToggleButton.jsx';
+import MeasureUtil from '../../Util/MeasureUtil/MeasureUtil';
+import MapUtil from '../../Util/MapUtil/MapUtil';
+
+import './MeasureButton.less';
+
+/**
+ * The MeasureButton.
+ *
+ * @class The MeasureButton
+ * @extends React.Component
+ */
+class MeasureButton extends React.Component {
+
+  /**
+   * The className added to this component.
+   *
+   * @type {String}
+   * @private
+   */
+  className = 'react-geo-measurebutton';
+
+  /**
+   * Currently drawn feature.
+   *
+   * @type {OlFeature}
+   * @private
+   */
+  _feature = null;
+
+  /**
+   * Overlay to show the measurement.
+   *
+   * @type {ol.Overlay}
```

Comment:
Plz use either dot or camelcase notation for ol classes. `ol.overlay` vs `OlOverlay`. I'd prefer the second one as we use it like this in react-geo.

Reply:
I prefer your suggestion too -> adjusted in e9f7173